src/python/tests/core/bot/tasks/fuzz_task_test.py

```diff
@@ -1350,6 +1350,8 @@ def setUp(self):
     os.environ['FUZZ_INPUTS_DISK'] = '/fuzz-inputs-disk'
     os.environ['BUILD_DIR'] = '/build_dir'
     os.environ['MAX_TESTCASES'] = '2'
+    os.environ['AUTOMATIC_LABELS'] = 'auto,auto1'
+    os.environ['AUTOMATIC_COMPONENTS'] = 'auto,auto1'
```

Comment:
Should probably have different value for component than labels to catch the bug highlighted above.

Reply:
Done!